src/Jobs/Recruit.Vacancies.Jobs/appSettings.Development.json

```diff
@@ -22,7 +22,8 @@
     "AzureWebJobsStorage": "UseDevelopmentStorage=true",
     "Redis": "localhost:6379",
     "MongoDb": "mongodb://dbadmin:changeme@localhost:27017/admin/?ssl=false&authMechanism=SCRAM-SHA-1",
-    "QueueStorage": "UseDevelopmentStorage=true"
+    "QueueStorage": "UseDevelopmentStorage=true",
+    "VacancyAnalyticEventsSqlDbConnectionString": "Data Source=127.0.0.1,1401;Initial Catalog=das-vacancy-analytic-events;User Id=sa;Password=Ozzyscottypassword1;MultipleActiveResultSets=True"
```

Comment:
> MultipleActiveResultSets=True 

Is this required?

Reply:
Not required but this is a local connection string so it doesn't matter.